components/dataflow/predict/component.yaml

```diff
@@ -0,0 +1,31 @@
+name: Predict using TF on Dataflow
+description: |
+  Runs TensorFlow prediction on Google Cloud Dataflow
+  Input and output data is in GCS
+inputs:
+  - {name: Data file pattern,   type: {GcsPath: {data_type: CSV}},        description: 'GCS or local path of test file patterns.'}
```

Comment:
Could we use GCSPath to be exact same with the type names in dsl/types.py

Reply:
Since adding the properties for the GCSPath confuses users(parameter types vs. artifact types), Could we drop the properties for the GCSPath? 
+cc @neuromage

Reply:
> Since adding the properties for the GCSPath confuses users

It's the opposite for me. It took me a lot of time to look through the source code to find what's the expected format for every GCSPath.

Reply:
> Could we use GCSPath to be exact same with the type names in dsl/types.py

You mean the casing? Sure, I'll change it.

Reply:
> Could we use GCSPath to be exact same with the type names in dsl/types.py

Done.